conf/rabbit-definition.conf

```diff
@@ -0,0 +1,35 @@
+{
+  "users": [
+    {
+      "name": {{ .Values.rabbitmq.auth.username | quote }},
+      "password": {{ .Values.rabbitmq.auth.password | quote }},
+      "tags": "administrator"
+    }
+  ],
+  "permissions": [
+    {
+      "user": {{ .Values.rabbitmq.auth.username | quote }},
+      "vhost": {{ .Values.rabbitmq.ingress.path | quote }},
+      "configure": ".*",
+      "write": ".*",
+      "read": ".*"
+    }
+  ],
+  "vhosts": [
+    {
+      "name": "/"
+    }
+  ],
+  "policies": [
+    {
+      "vhost":"/",
+      "name":"ha",
+      "pattern":"",
+      "definition": {
+        "ha-mode":"all",
+        "ha-sync-mode": "automatic",
+        "ha-sync-batch-size":10
+      }
+    }
+  ]
+}
```

Comment:
please include a newline at the end of file as Github suggests

Reply:
I have added an extra line to the conf file